docs/source/conf.py

```diff
@@ -475,6 +475,13 @@
     "get_weight_qspec",
     "propagate_annotation",
     "register_annotator",
+    # torch.ao.quantization.experimental.apot_utils
```

Comment:
Why are there new entries to this file? We should not have any new entry added to this file.